benchmarks/generate_data_footer_benchmark.py

```
@@ -0,0 +1,570 @@
+    schema = {
+        "columns": [
+            {"name": f"COLUMN_{i}", "type": col_type}
+            for i, col_type in enumerate(types)
+        ]
+    }
+
+    with open(file_path, mode="w") as schemafile:
+        schemafile.write(json.dumps(schema))
+
+
+def write_all_encodings_to_csv():
```

Ruff (0.11.9)

Missing return type annotation for public function `write_all_encodings_to_csv`

Add return type annotation: `None`

(ANN201)